ui/src/components/DashboardGlobalHealth.js

```diff
@@ -78,89 +63,100 @@ const DashboardGlobalHealth = () => {
   const platformStatus = highestAlertToStatus(platformHighestSeverityAlert);
 
   return (
-    <GlobalHealthContainer>
-      <div className="datacenter">
-        <SpacedBox ml={12} mr={12}>
-          <PlatformStatusIcon>
-            <StatusWrapper status={platformStatus}>
-              <StatusIcon status={platformStatus} name="Datacenter" />
-            </StatusWrapper>
-          </PlatformStatusIcon>
-        </SpacedBox>
+    <AppContainer.OverallSummary>
+      <GlobalHealthContainer>
```

Comment:
We could simplify the GlobalHealthContainer by using the Stack component.

```js
<Stack gap="r32" withSeparators >
  <Box flex={0.4} />
  <Box flex={0.4} />
  <Box flex={0.4} />
</Stack>
```

You can also find the example in core-ui stories:
https://scality.github.io/core-ui/?path=/docs/components-navigation-layout--layout-2-simplest

Reply:
Nope, the Stack component doesn't allow to set separator width. And I don't see the point of a separate component which does such a trivial action. Apparently we would need to extend it endlessly (more PRs, releases of core-ui and so on) just to have some thin meaningless layer

Reply:
> doesn't allow to set separator width
I don't think the separator width should be different than the one provided by the stack component. The whole idea of using the stack component is to unify it with others separators. 
The idea behind core-ui is to enforce a set of design rules so that such details don't need to be re-implemented in the app, and all the UIs got the same visual aspect. Even through this looks like an unsignificant details I would strongly push to use Stack separators and don't re-invent the wheel here. 
To me we don't need any changes on core-ui to add this separator. Through I will let @Cuervino confirm that the separation here should be the same as the other ones.

Reply:
Okay, given that you're strongly pushing for that option, can we make sure that the component actually fulfills the desired design? Right now it adds a whitespace (the width of which is font-size dependent), I see there's some min-height and min-width set in rem. Given that it's a separator, I don't see why we would want it to scale. The other problem is that the separator is outside the individual flex items, thus making it almost impossible to adjust its position in regards to other visual elements (you can go with flex values being something like 1.98 or whatever, but you see how absurd that is). And this situation and this screen are perfect example how you think you've made one component which serves all needs and in reality even in our existing pages - it doesn't. So that's why I suggested a compromise - at least add some mechanisms to extend the current behavior when and if needed. A sane default which covers most of the cases, and some props to tackle the more peculiar ones. Even in the epic-react all reusable components were built with extension in mind, every subcomponent and handler could be optionally replaced with a custom one

Here are two screenshots - one of my implementation and one of using Stack

![platform-dashboard-current](https://user-images.githubusercontent.com/106077432/198675718-c4ea9f1b-70d7-41fd-a7ad-cfa5ecc8f558.png)

![platform-dashboard-with-stack-component](https://user-images.githubusercontent.com/106077432/198675694-232d90ba-9cc4-43d6-91d0-217beb2962a4.png)

Reply:
My 2 cents:

- About **responsive/scalable**: agree, it's really not needed to have them scaling.
- I won't push too much for **alignement between separators and the sections below**, as the content of the Overall summary is not final, we might have new areas in the futur (activities, or something else). In fact, these separators are not really needed, and there are not used in others places of the UIs.